static/js/tippyjs.js

```diff
@@ -210,6 +210,14 @@ export function initialize() {
         },
     });
 
+    delegate("body", {
+        target: ".emojiset_choices > i",
+        appendTo: () => document.body,
+        onHidden(instance) {
+            instance.destroy();
+        },
+    });
```

Comment:
Can you add a comment documenting why this one needs custom behavior?

Reply:
At this point I don't remember why I put it there, and it seems like things work fine without it so I will take it out. 🤷 thanks!